module/os/linux/zfs/zpl_file_range.c

```diff
@@ -155,8 +156,18 @@ zpl_remap_file_range(struct file *src_file, loff_t src_off,
 	if (len == 0)
 		len = i_size_read(file_inode(src_file)) - src_off;
 
-	return (__zpl_clone_file_range(src_file, src_off,
-	    dst_file, dst_off, len));
+	ret = __zpl_clone_file_range(src_file, src_off,
+	    dst_file, dst_off, len);
+
+	/*
+	 * ENOTTY: see explanation zfs_clone_range.
+	 * This appears when something is not matching
+	 * the requirements of the filesystem.
+ 	 */
+	if (ret == ENOTTY)
```

Comment:
`-ENOTIFY` should be the same for the other places as well.

Reply:
@ixhamza can you please check it again. I changed the code and this is now outdated